osu.Game/Overlays/Comments/ReportCommentPopover.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.UserInterfaceV2;
+using osuTK;
+
+namespace osu.Game.Overlays.Comments
+{
+    public class ReportCommentPopover : OsuPopover
+    {
+        private readonly Action<CommentReportReason, string> action;
+        private LabelledEnumDropdown<CommentReportReason> reason = null!;
+        private LabelledTextBox info = null!;
+        private ShakeContainer shaker = null!;
+
+        public ReportCommentPopover(Action<CommentReportReason, string> action)
+        {
+            this.action = action;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            Child = new FillFlowContainer
+            {
+                Direction = FillDirection.Vertical,
+                Width = 500,
+                AutoSizeAxes = Axes.Y,
+                Spacing = new Vector2(7),
+                Children = new Drawable[]
+                {
+                    reason = new LabelledEnumDropdown<CommentReportReason>
+                    {
+                        Label = "Reason"
+                    },
+                    info = new LabelledTextBox
+                    {
+                        Label = "Additional info",
+                    },
```

Comment:
Use localised web strings instead (same applies everywhere a string is used for display in this PR, when possible). See `UsersStrings.Report*`.